src/lib/validationRules/validContactsField/validContactsField.test.ts

```diff
@@ -0,0 +1,105 @@
+//
```

Comment:
The copyright header is duplicated multiple times here, there is probably something wrong with the linting rule.

Reply:
Oh no! Just force-pushed a fix.